applications/asset_tracker_v2/overlay-lwm2m.conf

```diff
@@ -26,24 +26,64 @@ CONFIG_LWM2M_RW_SENML_CBOR_RECORDS=45
 CONFIG_ZCBOR=y
 CONFIG_ZCBOR_CANONICAL=y
 
-# Enable TLS session caching to prevent doing a full TLS handshake for every send.
+# Enable DTLS Connection Identifier and TLS session caching
+# to prevent doing a full TLS handshake for every send.
 CONFIG_LWM2M_TLS_SESSION_CACHING=y
+CONFIG_LWM2M_DTLS_CID=y
 
-# Set uptime to once every 12 hours.
+# Set uptime to 12 hours.
+# Update every 60 minutes (3600 s)
 CONFIG_LWM2M_ENGINE_DEFAULT_LIFETIME=43200
+CONFIG_LWM2M_UPDATE_PERIOD=3600
+CONFIG_LWM2M_SECONDS_TO_UPDATE_EARLY=60
 
 # LwM2M Queue Mode
 CONFIG_LWM2M_QUEUE_MODE_ENABLED=y
-CONFIG_LWM2M_RD_CLIENT_SUSPEND_SOCKET_AT_IDLE=y
+CONFIG_LWM2M_RD_CLIENT_STOP_POLLING_AT_IDLE=y
 # Sets the duration that the lwm2m engine will be polling for data after transmission before
 # the socket is closed.
 CONFIG_LWM2M_QUEUE_MODE_UPTIME=30
 
-## Set Requested Active Time (RAT) to 60 seconds. Preferably a little bit longer than the
-## configured LWM2M_QUEUE_MODE_UPTIME. Due to NAT/firewall UDP connections are usually
-## closed within 30-60 seconds so there is in general no point in setting a longer
-## Queue mode uptime / LTE PSM active time.
-CONFIG_LTE_PSM_REQ_RAT="00100001"
+# Configure PSM mode
+CONFIG_LTE_PSM_REQ=y
+# Request periodic TAU of 3600 seconds (60 minutes)
+CONFIG_LTE_PSM_REQ_RPTAU="00000110"
+
+# Set Requested Active Time (RAT) to 30 seconds. Preferably same as the
+# configured LWM2M_QUEUE_MODE_UPTIME. Due to NAT/firewall UDP connections are usually
+# closed within 30-60 seconds so there is in general no point in setting a longer
+# Queue mode uptime / LTE PSM active time.
+CONFIG_LTE_PSM_REQ_RAT="00001111"
+
+# Request eDRX mode
+CONFIG_LTE_EDRX_REQ=y
+
+# Requested eDRX cycle length for LTE-M and Nb-IoT
+# This should be fine-tuned for the network and the chosen server.
+# Lowest value is  the most responsive, but uses more energy during the active eDRX period.
+# Longer period may cause more CoAP packet drops on server requests.
+# "0000" is 5.12 s
+# "0001" is 10.24 s
+# "0010" is 20.48 s.
+CONFIG_LTE_EDRX_REQ_VALUE_LTE_M="0000"
+CONFIG_LTE_EDRX_REQ_VALUE_NBIOT="0000"
+
+# Request Paging time window of 1.28 seconds for LTE-M
+CONFIG_LTE_PTW_VALUE_LTE_M="0000"
+
+# Request Paging time window of 2.56 seconds for NB-IoT
+CONFIG_LTE_PTW_VALUE_NBIOT="0000"
```

Comment:
**question:** same `0000` here is 2.56 seconds, and in line 72 it's 1.28 seconds?

Reply:
Correct.

These values are not one-to-one between LTE-M and NB-IoT. Please see:
https://docs.nordicsemi.com/bundle/ref_at_commands/page/REF/at_commands/nw_service/cedrxs_set.html